### DISCUSSION TITLE
Show test execution lifecycle in console log during parallel execution

### DISCUSSION THREAD

**Discussion by alexnot95** [lyndsey-ferguson/fastlane-plugin-test_center#334] [Q&A] [answered]
Hi, @lyndsey-ferguson I have a question: How can we print test execution in Jenkins console log while we run for ex. on 3 parallel simulators?

For now, if I will run tests on 3 parallel simulators I can't see any info about how tests are going, I see info only at the end and I can't see any info in the console during execution, it looks like this:
![Screenshot 2021-03-11 at 17 54 28](https://user-images.githubusercontent.com/37154209/110815144-dc6e8a00-8292-11eb-923d-cbd0f2aa1dee.png)

And If I will run on 1 simulator I can see info about each test execution in console:
<img width="423" alt="Screenshot 2021-03-11 at 17 11 53" src="https://user-images.githubusercontent.com/37154209/110815246-f4460e00-8292-11eb-9792-cb21c36dfb06.png">

**Comment by lyndsey-ferguson** [comment, marked as answer]
Hi @alexnot95 currently, you cannot print the execution of each of the workers. I had to pipe the logs of each worker to separate logs because multiple threads writing to the same log ends up writing text over each other and it becomes unreadible.

The plugin can be re-worked to have a central log printer that parses the logs from each worker, but that will take time and it is not the top priority right now. I have this issue request that seems to be exactly what you're looking for #294.
